### ISSUE TITLE
Installation instructions for Mac outdated?

### ISSUE THREAD

**Issue by onestream** [partouf/OBSInfoWriter#58]
Hi, I used to have InfoWriter working, but with the update to Mac OS Sonoma 14.0 and with OBS Studio 29.1.3 (64 bit) I can't get it to work.

The installation instructions for Mac says:

1) Right-click on OBS in Applications and choose "Show package contents"
2) Place the libOBSInfoWriter.so file (from the libOBSInfoWriter.OSX.zip file) in the plugins folder

However, OBSInfoWriter-2.3 uses an installation .pkg which seems to run fine, which is not in sync with the installation instructions (?)

After running the OBSInfoWriter-1.0.0-macos-arm64.pkg found in the OBSInfoWriter v2.3 release, I do not find any InfoWriter Source to add to "1 scene".

What am I missing?

**Comment by partouf** [comment]
> However, OBSInfoWriter-2.3 uses an installation .pkg which seems to run fine, which is not in sync with the installation instructions (?)
> 
> After running the OBSInfoWriter-1.0.0-macos-arm64.pkg found in the OBSInfoWriter v2.3 release, I do not find any InfoWriter Source to add to "1 scene".
> 
> What am I missing?

Yes, I wrote something about that on the OBS forum for the release, I thought it was just because I have an old OSX version, but perhaps it will work for you:

```
For Mac: There's an installer, but it won't actually work (at least not for me) until you copy
Code:
/Library/Application Support/obs-studio/plugins/OBSInfoWriter.plugin
to the package contents of OBS
```

**Comment by onestream** [comment]
Thank you for the help! Your method seemed very promising, but when trying it now and (re)start OBS I still find no "InfoWriter" which I can add to my standard scene. This step is still necessary?

**Comment by partouf** [comment]
> Thank you for the help! Your method seemed very promising, but when trying it now and (re)start OBS I still find no "InfoWriter" which I can add to my standard scene. This step is still necessary?

Yes that should still be necessary.

Can you check your logging and see if it's at least attempted to being loaded?

**Comment by partouf** [comment]
I have attempted to make a new build for OBS29, in case that's the problem. https://github.com/partouf/OBSInfoWriter/releases/tag/2.5.1

I have not tested it myself yet

**Comment by onestream** [comment]
Yes - this worked - thank you so much! I did also follow your instructions to plonk ~/Library/Application Support/obs-studio/plugins/OBSInfoWriter.plugin into place (this seems to be necessary).

I used the 'macos-arm64.pkg' version before, but since only 'macos-universal.pkg' was available now I used this. Perhaps this was the error.
